custom-connectors/GroupMgr/apiDefinition.swagger.json

```diff
@@ -0,0 +1,325 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GroupMgr",
+    "description": "GroupMgr is a tool used to manage your Office 365 groups.",
+    "version": "1.0",
+	"contact": {
+      "name": "GroupMgr Helpdesk",
+      "url": "https://www.vanroey.be/support/",
+      "email": "support@groupmgr.com"
+    }
+  },
+  "host": "vragroupmanagerapprovalflow.azurewebsites.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/webhookrequest": {
+      "x-ms-notification-content": {
+        "description": "Caught group creation",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "DisplayName": {
+              "type": "string",
+              "description": "The display name of the group.",
+              "title": "Display name"
+            },
+            "GroupType": {
+              "type": "string",
+              "description": "The group type.",
+              "title": "Type"
+            },
+            "Email": {
+              "type": "string",
+              "description": "The groupmail.",
+              "title": "Email"
+            },
+            "Owners": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "description": "The emails of all the owners.",
+                "title": "Owners emails"
+              },
+              "description": "Owners"
+            },
+            "Members": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "description": "The emails of all the members.",
+                "title": "Members emails"
+              },
+              "description": "Members"
+            },
+            "isPublic": {
+              "type": "boolean",
+              "description": "If the group is public.",
+              "title": "Is public",
+              "enum": [
+                "",
+                "true",
+                "false"
+              ]
+            },
+            "CreateTeam": {
+              "type": "boolean",
+              "description": "If the group should create a Team.",
+              "title": "Create a Team",
+              "enum": [
+                "",
+                "true",
+                "false"
+              ]
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@Jwaegebaert I would recommend you to replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.